src/main/java/com/bajdas/rest_shop/notification/NotificationSpawner.java

```diff
@@ -0,0 +1,48 @@
+package com.bajdas.rest_shop.notification;
+
+import com.bajdas.rest_shop.model.ClientTransactionDto;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import software.amazon.awssdk.services.sns.SnsClient;
+import software.amazon.awssdk.services.sns.model.PublishRequest;
+
+@Component
+@Slf4j
+public class NotificationSpawner implements Observer {
```

Comment:
Jezeli odpowiedzialnosc tej klasy to tylko informowanie o udanej transakcji to bym zmienil nazwe